torch/quantization/fx/quantize.py

```diff
@@ -577,15 +589,40 @@ def convert(self, quantizer, node, load_arg, debug=False):
                         'expect bias provided as a keyword argument when it is not a positional argument'
                     bias = kwargs['bias']
                     kwargs.pop('bias')
-                prepack_args = [weight, bias]
+                prepack_args = (weight, bias)
                 packed_weight = quantizer.quantized_graph.create_node(
                     'call_function', torch.ops.quantized.linear_prepack, prepack_args, {})
                 # construct dynamic linear input
                 linear_input = load_arg(quantized=False)(self.linear_node.args[0])
-                qdynamic_linear_args = [linear_input, packed_weight]
+                qdynamic_linear_args = (linear_input, packed_weight)
                 return quantizer.quantized_graph.create_node(
                     'call_function', torch.ops.quantized.linear_dynamic, qdynamic_linear_args, kwargs)
 
+
+# Patterns for weight prepack op folding
+# Base Weight Prepack Folding Pattern Handler
+class FoldingHandler(ABC):
```

Comment:
makes sense, would this be used in later PRs?  Wasn't sure how it's used here.

Reply:
this will be removed, I thought I need this in the beginning, but turns out we don't